historical_system_profiles/notification_service_interface.py

```diff
@@ -76,16 +76,9 @@ class EventDriftBaselineDetected(_NotificationEventBase):
     """Message for drift-baseline-detected notification event"""
 
     def __init__(
-        self,
-        account_id,
-        insights_id,
-        inventory_id,
-        system_check_in,
-        display_name,
-        tags=[],
+        self, account_id, inventory_id, system_check_in, display_name, tags=[],
```

Comment:
Unrelated to these changes, but there is a list instance as a default value for `tags`. List is mutable, and in case of Python having it as a default value would mean that the instance is shared between calls. Any mutation of this default value would affect subsequent calls. Better to replace it with a tuple (in separate PR): `tags=()`.

Reply:
This is really good catch, @vkrizan. I am creating a jira ticket to get these checked out.